### DISCUSSION TITLE
rraytable里的组件 在点击的时候 如何获取当前行的数据 版本是1.x的

### DISCUSSION THREAD

**Discussion by wangzyeeeeee** [alibaba/formily#1234] [Q&A 1.x]
如题

**Comment by janryWang** [comment]
你说的是什么点击，删除？还是移动

**Reply by wangzyeeeeee** [reply]
组件的点击 不是在操作行列的点击；
 例如： 点击 下图的的input 能够获取到当前行的数据 
![image](https://user-images.githubusercontent.com/67303799/114985783-cb362000-9ec5-11eb-81d3-e2a5a3028133.png)

**Reply by janryWang** [reply]
1.x ArrayTable没有暴露获取索引的能力，自己封装自定义组件吧，拿到索引之后，getFieldState拿下数据就行了
